recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -1,10 +1,29 @@
-SRC_URI += "file://weston.ini"
+inherit systemd
+
+SRC_URI += "file://weston.ini \
+            file://weston-start \
+            file://weston.service \
+            file://weston.path \
```

Comment:
It's a path triggered unit. Thing is that /dev/dri/card0 may not exist by the time Weston starts. We need to ensure that this path exists, and that is done with a path unit. See below for ConditionPathExists.